src/codewhisperer/util/telemetryHelper.ts

```diff
@@ -317,43 +317,44 @@ export class TelemetryHelper {
         if (e2eLatency < 0) {
             e2eLatency = performance.now() - session.invokeSuggestionStartTime
         }
-
-        client
-            .sendTelemetryEvent({
-                telemetryEvent: {
-                    userTriggerDecisionEvent: {
-                        sessionId: sessionId,
-                        requestId: this.sessionDecisions[0].codewhispererFirstRequestId,
-                        customizationArn: selectedCustomization.arn === '' ? undefined : selectedCustomization.arn,
-                        programmingLanguage: {
-                            languageName: runtimeLanguageContext.toRuntimeLanguage(
-                                this.sessionDecisions[0].codewhispererLanguage
-                            ),
+        if (this.sessionDecisions[0].codewhispererSessionId) {
```

Comment:
This condition is not checked before line 273. Is that correct?

Reply:
+1, should we combine this one with the check on ln 256?